src/bebone/assets/image/image_painter.cpp

```diff
@@ -50,6 +69,17 @@ namespace bebone::assets {
         }
     }
 
+    template<typename Color>
+    void ImagePainter<Color>::paint_square(const Vec2i& pos, const Vec2i & size, const Color& color) {
```

Comment:
Change `const Vec2i & size` to `const Vec2i& size`

Reply:
Ok boss !